bluebrain/repo-patches/packages/steps/package.py

```diff
@@ -111,9 +112,9 @@ def cmake_args(self):
             args.append("-DENABLE_CODECOVERAGE:BOOL=True")
 
         if "+distmesh" in spec:
-            args.append("-DUSE_DISTRIBUTED_MESH:BOOL=True")
+            args.append("-DSTEPS_USE_DIST_MESH:BOOL=True")
         else:
-            args.append("-DUSE_DISTRIBUTED_MESH:BOOL=False")
+            args.append("-DSTEPS_USE_DIST_MESH:BOOL=False")
```

Comment:
Should this be guarded by Steps version? Will this work when we build 3.6?

Reply:
This option was introduced in the version we are about to release (4.0). It is not used by 3.6